components/datetime/src/provider/calendar/skeletons.rs

```diff
@@ -43,3 +43,57 @@ impl TryFrom<&str> for SkeletonV1 {
         }
     }
 }
+
+#[cfg(feature = "datagen")]
```

Comment:
nit: comment explaining why this is doing a weirder thing with BakedDateSkeletonPattern